rpm/aardvark-dns.spec

```diff
@@ -53,6 +59,31 @@ BuildRequires: rust-srpm-macros
 Forwards other request to configured resolvers.
 Read more about configuration in `src/backend/mod.rs`.
 
+# Only intended to be used for gating tests
+# End user usecases not supported
```

Comment:
should this be part of the description? So end user have actually a chance to read it.